src/components/remotecontrol/remotecontrol.js

```diff
@@ -215,7 +215,7 @@ define(["browser", "datetime", "backdrop", "libraryBrowser", "listView", "imageL
                 context.querySelector(".sendTextSection").classList.add("hide");
             }
 
-            if (!currentPlayer.isLocalPlayer) {
+            if (-1 != supportedCommands.indexOf("MoveUp" || "MoveDown" || "MoveRight" || "MoveRight" || "MoveLeft" || "Select" || "Back" || "ToggleContextMenu" || "GoToSearch" || "GoToSettings" || "GoHome") && !currentPlayer.isLocalPlayer) {
```

Comment:
are you sure this is doing what you think it does? :)
to me it seems you're actually only checking for `.indexOf("MoveUp")` ;)

Reply:
Hm, yes it seems you're right... 😒
Do you have a hint for me how do solve it without checking for them seperate?
Or should I just keep the collapse visible with only disabled buttons?

I found out that I dont have to hide each button, they get disabled if not supported:
![image](https://user-images.githubusercontent.com/14938497/72170301-ebc51f80-33d0-11ea-9a49-cd8d7e2c66f3.png)